WebViewControl.Avalonia/WebView.Avalonia.cs

```diff
@@ -92,7 +92,7 @@ partial class WebView : BaseControl {
         /// <paramref name="isSystemEvent">True if is a system focus event, or false if is a navigation</paramref>
         /// </summary>
         protected virtual bool OnSetFocus(bool isSystemEvent) {
-            var focusedElement = TopLevel.GetTopLevel(this).FocusManager.GetFocusedElement();

```

Comment:
Can you add a comment for future generations on why we need this? 🙂

Reply:
Added